packaging/build_packages_rpm.sh

```diff
@@ -46,7 +46,7 @@ case ${RELEASE} in
   6*) el6_install;;
   7*) yum -y install https://rhel7.iuscommunity.org/ius-release.rpm;;
 esac
-rpm --import /etc/pki/rpm-gpg/IUS-COMMUNITY-GPG-KEY
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-IUS-7
```

Comment:
I assume this will fail for el6 (I bet theres a /etc/pki/rpm-gpg/RPM-GPG-KEY-IUS-6 key?), you probably need to add this into the el6_install function and add a el7_install function as well

Reply:
check the rpm gpg import on rhel-6. refactoring the current script to accommodate the changes